Week_01_Basics/homework-portfolio/portfolio.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<head>
+	<link href="./style.css" rel="stylesheet">
+	<title>Portfolio | Karmaela by Michelle Zhang</title>
+	<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+</head>
+<body>
+	<a name="back">
```

Comment:
I was trying to create an anchor at the top to return to the top. I thought maybe reloading the full page would take longer? Thinking out loud nowm would it work if I try <a href="#">?